rds-postgres/admin-login/rotation/lambda_function.py

```diff
@@ -289,9 +294,13 @@ def dict_to_url(secret):
     Returns:
         url: DATABASE_URL-style string
     """
+    if replica:
+        host = secret['host']
+    else:
+        host = REPLICA_HOST
```

Comment:
Updated this rotation script to use the env var `REPLICA_HOST`, rather than passing the host address of the replica DB in two ways, in response to @jferris's comment [here](https://github.com/thoughtbot/terraform-aws-databases/pull/7#pullrequestreview-1179479280).